tools/bag_processing/scripts/rosbag_topic_filter.py

```diff
@@ -135,18 +135,16 @@ def topic_filter(verbose, inbag_path, accept_patterns, reject_patterns, outbag_p
     parser.add_argument(
         "-a",
         "--accept",
-        nargs="?",
+        nargs="*",
```

Comment:
Hm. Your change seems intuitively right to be, but (1) I feel like it was probably working before (because the examples are not arbitrary but represent a way I actually ran it at one time), and (2) I find `argparse` semantics sometimes counter-intuitive.

(But by all means, if you tested it both ways and this fixes a problem, thanks for finding it!)

Reply:
yup, tested it before and didn't work, then made the change and worked